src/storyblok/Hero.astro

```diff
@@ -1,19 +1,47 @@
 ---
 import { storyblokEditable } from "@storyblok/astro";
+import { Image } from "astro:assets";
 import StoryblokComponent from "@storyblok/astro/StoryblokComponent.astro";
+import type { ButtonStoryblok } from "../../storyblok-component-types";
```

Comment:
**Q**: Is there any alias in Astro we can use to avoid relative paths? Asking because in Nuxt we use @ or ~ and it is much comfy 🤩

Reply:
Yup! But for `.astro` files, so not for generated type files...
[https://docs.astro.build/en/guides/aliases/](https://docs.astro.build/en/guides/aliases/)